tailwind.config.js

```diff
@@ -87,6 +89,8 @@ module.exports = {
     fontFamily: {
       sans: ["NEXT Book", "Arial", "Helvetica", "sans-serif"],
       mono: ["Source Code Pro", "Courier", "monospace"],
+      manrope: ["Manrope", "sans-serif"],
```

Comment:
`sans` → `sans_next`
`mono` → `mono_source_code`
`manrope` → `sans`
`jetbrains_mono` → `mono`

Reply:
finished in newest fixup, can you check it if it is satisfactory @kennethkalmer ?